src/main/java/io/cryostat/targets/TargetConnectionManager.java

```diff
@@ -85,6 +86,19 @@ public class TargetConnectionManager {
     private final Duration failedBackoff;
     private final Duration failedTimeout;
 
+    @ConfigProperty(name = ConfigProperties.URI_RANGE, defaultValue = "PUBLIC")
```

Comment:
I think you can skip the `defaultValue` here since there is also a default value already given in `application.properties`